Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -92,18 +92,20 @@ void PPCSymbolDB::AddKnownSymbol(u32 startAddr, u32 size, const std::string& nam
 Common::Symbol* PPCSymbolDB::GetSymbolFromAddr(u32 addr)
 {
   auto it = m_functions.lower_bound(addr);
-  if (it == m_functions.end())
-    return nullptr;
-
-  // If the address is exactly the start address of a symbol, we're done.
-  if (it->second.address == addr)
-    return &it->second;
 
-  // Otherwise, check whether the address is within the bounds of a symbol.
+  if (it != m_functions.end())
+  {
+    // If the address is exactly the start address of a symbol, we're done.
+    if (it->second.address == addr)
```

Comment:
This if statement can be combined with the above one: `it != m_functions.end() && it->second.address == addr`. Feel free to ignore this comment.

Reply:
I think it's a bit more readable this way since the one below needs to be two separate statements, so having this one be two makes it a bit more consistent.